ioutil/write_serialized_file.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package ioutil
+
+import (
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+)
+
+// WriteSerializedFile writes (or overwrites) `data` into `filename`.
+// If `filename` doesn't exist, it's created with `perm` permissions.
+// If `filename` does exist, the data is first overwritten to the
+// file, and then the file is truncated to the length of the data.  If
```

Comment:
The `File.Write()` call should just extend the file in that case, no problem.  (Just like if the file was brand new and empty.)